druid/src/widget/flex.rs

```diff
@@ -652,6 +680,15 @@ impl<T: Data> Widget<T> for Flex<T> {
                 }
                 Child::FixedSpacer(kv, calculated_siz) => {
                     *calculated_siz = kv.resolve(env);
+                    *calculated_siz = if *calculated_siz >= 0.0 {
+                        *calculated_siz
+                    } else {
+                        tracing::warn!(
+                            "Length provided to fixed spacer was les than 0: {}",
+                            *calculated_siz
+                        );
+                        0.0
+                    };
```

Comment:
I would probably just write this as,

```suggestion
                    if *calculated_siz < 0.0 {
                        tracing::warn!("Length provided to fixed spacer was less than 0");
                    }
                    *calculated_size = calculated_size.max(0.0);
```